test/Main.purs

```diff
@@ -224,3 +231,20 @@ main = runTest do
         sa = resize a
         sa' = matrix22 1.0 4.0 3.0 0.0
       equal sa' sa
+
+    test "findMaxIndex" do
```

Comment:
Adding a basic test for an example here might also be helpful. 
Sth. like `equal 10 $ findMaxIndex (1 +> 10 +> 3 +> Vec.empty)`

Reply:
ok, now I understand. Sure I'll add one..